src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSessionsPageUiTest.java

```diff
@@ -620,6 +623,27 @@ private void testPublishAction() throws Exception {
         ______TS("PUBLISHED: publish link hidden");
         feedbackPage = getFeedbackPageForInstructor(idOfInstructorWithSessions);
         feedbackPage.verifyPublishLinkHidden(courseId, sessionName);
+
+        // Test that the resend published link button can be clicked and the form can be cancelled
```

Comment:
Use `______TS("Resend published link button can be clicked and the form can be cancelled")` instead of the comment

Reply:
Actually you can put one ` ______TS("resend link action: PUBLISHED feedback session");` and then leave the comments as they are